README.md

```diff
@@ -1,9 +1,9 @@
-[![](https://img.shields.io/pypi/v/dsp-tools.svg)](https://pypi.org/project/dsp-tools/) 
-[![](https://img.shields.io/pypi/l/dsp-tools.svg)](https://pypi.org/project/dsp-tools/) 
+[![DSP-TOOLS version on PyPI](https://img.shields.io/pypi/v/dsp-tools.svg)](https://pypi.org/project/dsp-tools/) 
+[![Supported Python versions](https://img.shields.io/pypi/pyversions/dsp-tools.svg)](https://pypi.org/project/dsp-tools/)
```

Comment:
I thought it would be nice to display the supported python versions in the badges on top. For this, pyproject.toml must contain the appropriate versions.

Then, I slightly rearranged our badges:
- added an explanatory alt text for all images (mainly for us)
- got rid of 2 badges (markdownlint and markdown-link-validator), because I think they are less important. It's better if the other (more important ones) stay out.